tests/test-pipenv.bsh

```diff
@@ -18,3 +19,39 @@ begin_test "Pipenv"
 
 )
 end_test
+
+# test 30_get-pipenv script without a container
+# Note this test assumes a system-wide python is available
```

Comment:
Code this assumption:

```
command -v "${PIPENV_PYTHON-python}" &>/dev/null || command -v python3 &>/dev/null || skip_next_test
```

- (Not sure it's worth caring about python2 anymore, I could go either way)
- Yes, I put the `PIPENV_PYTHON` in there even though `run_tests` would prevent you from setting that, because it is also allowed to run a test file directly, in which case `PIPENV_PYTHON` could be set

Reply:
Added